code/datums/emergency_calls/emergency_call.dm

```diff
@@ -83,7 +83,16 @@
 	if(SSticker?.mode?.waiting_for_candidates) //It's already been activated
 		return FALSE
 
-	picked_call.mob_max = rand(5, 15)
+	if(istype(src, /datum/game_mode/colonialmarines) ) //If we're fighting benos
+		var/xenos = length(GLOB.alive_xeno_list)
```

Comment:
There is a discrete proc in in gamemode code for doing what you're doing here.

Reply:
Well, what is it?

Reply:
count_humans_and_xenos()